interface-elements-for-desktop/articles/rich-text-editor/miscellaneous/insert-a-watermark.md

```diff
@@ -0,0 +1,53 @@
+# Insert a Watermark
+
+A watermark is a faded background image or text displayed behind document content. The WinForms Rich Text Editor allows you to create and remove watermarks. You can also print and export documents with watermarks to PDF.
```

Comment:
Isn't a watermark usually displayed over content?

Reply:
In the case of Word documents, watermarks are displayed behind content

Reply:
Ok, that is fine. Just making sure